packages/browser-logs/test/serialize-deserialize.test.ts

```diff
@@ -447,10 +445,10 @@ describe('serialize deserialize', () => {
     });
     expect(deserialized).to.be.a('string');
     expect(deserialized).to.include('my error msg');
-    expect(deserialized).to.include(`  at c (test${sep}__puppeteer_evaluation_script__:2:29)`);
-    expect(deserialized).to.include(`  at b (test${sep}__puppeteer_evaluation_script__:3:29)`);
-    expect(deserialized).to.include(`  at a (test${sep}__puppeteer_evaluation_script__:4:29)`);
-    expect(deserialized).to.include(`  at test${sep}__puppeteer_evaluation_script__:5:38`);
+    expect(deserialized).to.include(`  at c (test:2:29)`);
+    expect(deserialized).to.include(`  at b (test:3:29)`);
+    expect(deserialized).to.include(`  at a (test:4:29)`);
+    expect(deserialized).to.include(`  at test:5:38`);
```

Comment:
These changes look suspicious to me, but I'm not familiar with the logic they cover.

Reply:
@LarsDenBakker what do you think?